source/iterator/snapshot.go

```diff
@@ -0,0 +1,198 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iterator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/conduitio-labs/conduit-connector-oracle/coltypes"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/huandu/go-sqlbuilder"
+	"github.com/jmoiron/sqlx"
+)
+
+// Snapshot represents an implementation of a Snapshot iterator for Oracle.
+type Snapshot struct {
```

Comment:
please add briefly comments to fields.  For example
// orderingColumn Name of column what iterator using for sorting data.
orderingColumn

Reply:
Ok.